driver/src/main/java/org/kaazing/k3po/driver/internal/control/handler/ControlServerHandler.java

```diff
@@ -65,6 +68,7 @@
     private static final String ERROR_MSG_ALREADY_PREPARED = "Script already prepared\n";
     private static final String ERROR_MSG_ALREADY_STARTED = "Script has already been started\n";
 
+    private static Robot lastRobot;
```

Comment:
We have one ControlServerHandlers instance per connection. I have checked it in debug. So we need the static. 
If there was only one instance, everything will be messed up with multiple connections, as there are many non-static fields in the class. For example, channelClosedFuture will be completed by the first test and will be set from the start for all other tests.